samples/todo-app/database-api/MongoHelper.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MongoDB.Bson;
+using MongoDB.Driver;
+
+namespace DatabaseApi
+{
+    public static class MongoHelper
+    {
+        private static string _connectionString { get; set; }
+
+        private static string _database { get; set; }
+
+        private static string _collection { get; set; }
+
+        private static MongoClient _mongoClient;
+
+        public static void Initialize(string connectionString, string database, string collection)
+        {
+            Console.WriteLine("MongoHelper initialization start");
+            _database = database;
+            _collection = collection;
+            _mongoClient = new MongoClient(connectionString);
+            ValidateMongoConnection(_mongoClient, _database);
+            Console.WriteLine("MongoHelper initialization end");
+        }
+
+        private static async void ValidateMongoConnection(MongoClient mongoClient, string database)
+        {
+            var db = mongoClient.GetDatabase(database);
+
+            while (true)
+            {
+                try
+                {
+                    BsonDocument isMongoLive = await db.RunCommandAsync((Command<BsonDocument>)"{ping:1}");
+                    if (!isMongoLive.Contains("ok") || isMongoLive["ok"].AsDouble != 1.0)
+                    {
+                        Console.Error.WriteLine("Mongo connection dead! Shutting down.");
+                        Environment.Exit(1);
+                    }
+                }
+                catch (Exception e)
+                {
+                    Console.Error.WriteLine("Exception pinging Mongo:");
```

Comment:
nit: `.` instead of `:`

Reply:
This was code coming from the Bike Sharing sample that I reused as is: the string is ending with ":" because the exception is printed using the next console WriteLine. I updated the string so that everything is on the same line to make this clearer.